CONTRIBUTING.md

```diff
@@ -20,13 +22,13 @@
 
 ### Merge
 Once the pull request is reviewed, it can be merged to the develop branch
-- Go to the development branch:</br>```git checkout develop```
-- Merge your changes to the development branch:</br>```git merge <branch-name>```
-- Push the merge to github:</br>```git push```
+- Go to the development branch:<br>```git checkout develop```
```

Comment:
Do you also want to talk about how to do this from github itself as an alternative?  Or just leave this here as is.

I was thinking something like:

### Merge
Merges can be done two ways:
1. (Existing way via command line)
2. (Via the github WWW site)